addons/hr/models/hr.py

```diff
@@ -260,6 +264,12 @@ def _message_auto_subscribe_notify(self, partner_ids):
         # Do not notify user it has been marked as follower of its employee.
         return
 
+    @api.depends('address_home_id')
```

Comment:
Triggers can go deeper in relationship fields. Here we can be more specific and tell the trigger is address_home_id.parent_id . That way if someone hcanges the address_home_id street field address_home_has_company is not computed again.

I know it will have no impact in our case because it is an informative field used only in a form view but it is a good practice to do with computed fields.